integration_tests/test_model_download.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+
+from requests import HTTPError
+
+from kagglehub import model_download
+
+HANDLE = "keras/bert/keras/bert_base_en/2"
+
+
+class TestModelDownload(unittest.TestCase):
+    def test_model_versioned_succeeds(self):
+        expected_path = "kagglehub/models/keras/bert/keras/bert_base_en/2"
+        actual_path = model_download(HANDLE)
+        split_index = actual_path.find("kagglehub")
```

Comment:
The path where the file are saved is irrelevant.

I would simply assert on the files in the returned path:

```
path = model_download(HANDLE)

files = # list files at `path` (make sure you find all the file recursively).
self.assertEqual(["assets/tokenizer/vocabulary.txt", "config.json", "metadata.json", "model.weights.h5", "tokenizer.json"], files)
# TODO: Also assert that the file size > than zero.
# TODO: consider creating a utility test method maybe assertFiles() that do both.
```